ratelimits/src/index.ts

```diff
@@ -0,0 +1,71 @@
+import Limits from './limits.json'
+const limits: Record<string, any> = Limits // ugly solution to dynamically reference JSON properties
```

Comment:
```suggestion
type provider = keyof typeof Limits;
```

Reply:
couldn't sort the typing issue out here, wanted to be able to call `getRateLimit` with `tier` as a `string`